decidim-admin/spec/shared/manage_process_steps_examples.rb

```diff
@@ -97,7 +98,7 @@
     let!(:process_step2) { create(:participatory_process_step, participatory_process: participatory_process) }
 
     before do
-      visit decidim_admin.participatory_process_path(participatory_process)
+      visit current_path
```

Comment:
`current_path` is a capybara method, not a `let`. It's set once you visit any page. 

This is a shared example because 2 different profiles can do all these actions: admins and process admins.